src/applications/claims-status/actions/index.js

```diff
@@ -366,8 +376,18 @@ export function submitFiles(
                             Your submission is in progress. It can take up to 2
                             days for us to receive your files.
                           </span>
+                          {timezoneMitigationEnabled &&
```

Comment:
Non blocking, but worth noting that  this code is duplicated on lines 409 - 418. I'm going with the juice isn't worth the squeeze to extract this to a component given this is only used in two places and a temporary fix.